editor/components/inserter/group.js

```diff
@@ -74,13 +74,15 @@ export default class InserterGroup extends Component {
 	render() {
 		const { labelledBy, blockTypes } = this.props;
 
-		return <NavigableMenu
-			className="editor-inserter__category-blocks"
-			orientation="vertical"
-			aria-labelledby={ labelledBy }
-			cycle={ false }
-			onNavigate={ this.onNavigate }>
-			{ blockTypes.map( this.renderItem, this ) }
-		</NavigableMenu>;
+		return (
+			<NavigableMenu
```

Comment:
Thanks for the cleanup. Eventually, I hope to get to the point where you don't have to do this every time :)

Reply:
No worries :), it's almost therapeutic. We just need the robots to do more for us (more linting rules + autoformatters).